dace/transformation/dataflow/tasklet_fusion.py

```diff
@@ -0,0 +1,294 @@
+# Copyright 2019-2021 ETH Zurich and the DaCe authors. All rights reserved.
+""" Contains classes that fuse Tasklets """
+
+from typing import Any, Dict, Set
+import ast
+import dace
+import re
+from dace import dtypes, registry
+from dace.sdfg import nodes
+from dace.sdfg import utils as sdutil
+from dace.transformation import transformation as pm
+
+
+class ConnectorRenamer(ast.NodeTransformer):
```

Comment:
isn't this ASTFindReplace from dace.frontend.python.astutils?

Reply:
It seems that yes, but `ASTFindReplace` does more things that are not needed. Hmm, not sure if should change it or not.

Reply:
My tests do not pass after doing the change. I think `ASTFindReplace` returns after doing a single `ast.Name` replacement, while `ConnectorRenamer` will replace all `ast.Names`. I am not sure if this is intended or a bug but I don't think it concerns this PR.